tools/integration_tests/util/setup/setup.go

```diff
@@ -404,6 +405,30 @@ func AreBothMountedDirectoryAndTestBucketFlagsSet() bool {
 	return false
 }
 
+// Explicitly set the enable-hns config flag to true when running tests on the HNS bucket.
+func AddHNSFlagForHierarchicalBucket(ctx context.Context, storageClient *storage.Client) ([]string, error) {
+	attrs, err := storageClient.Bucket(TestBucket()).Attrs(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("Error in getting bucket attrs: %w", err)
+	}
+	if !attrs.HierarchicalNamespace.Enabled {
+		return nil, fmt.Errorf("Bucket is not Hierarchical")
+	}
+
+	var flags []string
+	mountConfig4 := config.MountConfig{
+		EnableHNS: true,
+		LogConfig: config.LogConfig{
+			Severity:        config.TRACE,
```

Comment:
Why is this needed?

Reply:
It is for integration tests. So in case of failure we can see logs.